.gitignore

```diff
@@ -110,5 +110,8 @@ result-*
 # emacs misc
 .dir-locals.el
 
+# vscode config
+.vscode
```

Comment:
Maybe have that in your local `~/.gitignore`? 👍

Reply:
I've removed it from the PR.